server/db/models/User.js

```diff
@@ -1,12 +1,13 @@
+require('dotenv').config();
 const {
   Model,
   DataTypes: { STRING, VIRTUAL },
 } = require('sequelize');
 const bcrypt = require('bcrypt');
 const jwt = require('jsonwebtoken');
+const stripe = require('stripe')(process.env.STRIPE_PK);
```

Comment:
small thing, I had it as "STRIPE_PUBLISHABLE_KEY" in the .env file so we'll just have to update what it's being called

Reply:
Okay, I'll change that to be consistent with yours and merge